docs/user_manual/array_runs.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "19. Array Runs"
+section: 19
+---
+
+
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Array Runs List" %}
+
+To get to the main Array Runs list page, click "Runs" under Array Scanning in the menu on the left side of the screen.
+This list includes all array runs in MISO. The toolbar at the top of the table includes many controls for working with
+array runs.
+
+You can find a similar list in the Array Runs section of the Edit Project page. The list on the Edit Project page only
+includes array runs that contain samples belonging to the project being viewed/edited. This list has the same controls
+as on the main Array Runs list page.
+
+{% include userman-toplink.md %}
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Creating an Array Run" %}
+
+Click the "Add" button in the toolbar at the top of the Array Runs list to get to the Create Array Run page. Here you
+can enter all of the array run details.
+
+To add an array to the run, enter the array's alias or serial number into the Array Search box and click the "Search"
+button. Matching arrays will appear in the dropdown below. Endure that the correct array is selected, then click the
+"Apply" button to confirm.
+
+When you are done, click the "Save" button at the top right to save the array run. After saving, you will be taken to the
+single Edit Array Run page, where you can further modify the array run.
+
+{% include userman-toplink.md %}
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Editing an Array Run" %}
+
+To get to the Edit Array Run page, click on the array run's ID or alias in the Array Runs list. The top section of this
+page is the same as on the Create Array Run page, and allows you to modify the array run details. Make any changes you
+would like, then click the "Save" button at the top right. The Samples section below shows all of the samples in the
+array if one has been added to the array run. Under that is another list containing the array run's change log.
```

Comment:
`if one has` -> `if any have`. Or, `The Samples section below shows all of the samples that have been added to the array run.`

Reply:
The first suggestion implies that more than one array can be added to the run, and the second implies that samples are added directly to the array run. Reworded to "If an array has been added to the array run, all of its samples are listed in the Samples section below." which should be clearer